spec/helpers/mock_project.rb

```diff
@@ -119,14 +119,37 @@ def bundle_exec(command, env = {})
       end
     end
 
+    TAPIOCA_DEPENDENCIES = T.let(
+      [
+        "diff-lcs",
+        "rbi",
+        "spoom",
+        "tapioca",
+        "thor",
+        "unparser",
+        "yard",
+        "yard-sorbet",
+      ],
+      T::Array[String],
+    )
+
     # Run a Tapioca `command` with `bundle exec` in this project context (unbundled env)
-    sig { params(command: String, enforce_typechecking: T::Boolean).returns(ExecResult) }
-    def tapioca(command, enforce_typechecking: true)
+    sig do
+      params(
+        command: String,
+        enforce_typechecking: T::Boolean,
+        exclude_tapioca_dependencies: T::Boolean,
+      ).returns(ExecResult)
+    end
+    def tapioca(command, enforce_typechecking: true, exclude_tapioca_dependencies: true)
```

Comment:
What do you think about passing the list of dependencies to ignore and allowing a default value?

```suggestion
    def tapioca(command, enforce_typechecking: true, exclude: TAPIOCA_DEPENDENCIES)
```

Reply:
I see the appeal but isn't this kind of strange given other flags aren't a keyword argument? I preferred the explicit intention of the boolean but don't have strong opinions.

Reply:
If we're allowing controlling the behavior of the exclude on top of the options already passed in the command string, we may as well pass the values to be used so the same approach can be reused in other tests that may need a special treatment.